src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const mult = days * 40;
+  let result = mult;
+
+  if (days >= 7) {
+    result = mult - 50;
+  } else if (days >= 3) {
```

Comment:
change the if else construction:
**[checklist] 4. Prefer if with return over if else to simplify later conditions.**

Reply:
Good point